### ISSUE TITLE
External servers not working

### ISSUE THREAD

**Issue by Harp3rDev** [titaniumnetwork-dev/Phoenix-Reborn#2]
So I don't know why but after deploying the app, viewing it goes to the local php proxy, and clicking on the other urls doesn't go anywhere else. This might be because all of the features aren't finished yet but idk.

edit: turns out I'm stupid, so every page works except for go home, which only takes me to the php proxy, I mainly used censordodge on rubyphoenix. If you guys add censordodge, that would be the greatest.

edit 3: There is censordodge, but i cant access it without typing the url (website.com/local.html) pressing go home takes me to the /local-2.html, and not a home page for to get to everything, or at least local.html

**Comment by LQ16** [comment]
Alright, this is an issue I forgot about and never actually fixed. Basically what you need to do is the following:

1. Create a .htaccess file
2. put the following line into it:  "DirectoryIndex index.html" (without quotes)
3. push it to the repository
4. redeploy
5. profit

**Comment by Harp3rDev** [comment]
Thanks, I will let you know if I have trouble doing it because I'm kinda new to github.

**Comment by Harp3rDev** [comment]
Thank you so much! It totally worked and now just in case ruby or diamond gets blocked, I have another! Side note: bigblockchevy.ml isn't working, as seen in this picture 
![image](https://user-images.githubusercontent.com/51060508/74161372-d409d280-4bec-11ea-8478-ba9a8009f20a.png) and audiosoundsystem.port0 is blocked at my school. Sorry to be sort of nitpicking since I know you guys have other things to be working on, but I would like to play blocked io games that don't run with censordodge or php proxy.

**Comment by LQ16** [comment]
we no longer maintain the external servers on this repo due to how frequently we change servers, your best bet is to take the servers from https://beep.meeps.us and put them on here.

**Comment by Harp3rDev** [comment]
Thanks once again

**Comment by Harp3rDev** [comment]
Well, I deployed my own node unblocker, so I am pretty much good for now. But then as for taking a server from beeps.meeps.us, I found https://cdn.unprecedented-novelist-improve-lovely.cf. But I don't know how it enters URLs since it is used as an iframe. Because in the node unblocker way I just made it go to ` window.location.href = https://aquaticlifenews.herokuapp.com/subject/${url};` and it works just fine, but I don't know how the $url goes in.
